### ISSUE TITLE
EPF usage huge difficulties

### ISSUE THREAD

**Issue by ceraolo** [PowerGrids/PowerGrids#164]
EPF would be a major improvement in PowerGrids, if develop branch is ever published.

However, I found it extremely difficult (if even possible) to use insofar, since I could not even envisage where to put my hands on.

I see three major issues:
**1. Examples.** 
Power grids has around 35 examples, but only one, very complex, with EPF: IEEE14busStaticNetworkEPF. This one is very difficult to understand (I could not find where to set powerFlow data). I think Powergrids should provide 2-3 much simpler examples. An example (to be improved) is discussed in the following point 2.
**2. PF-Transient continuity.** I opened on this ticket #151. It had some discussion, but did not result in an actual example in the library. In my comment https://github.com/PowerGrids/PowerGrids/issues/151#issuecomment-2495845145 I just mentioned how to reach effectively PF-transient continuity
**3.  UsersGuide.** In UsingPowerGrids, it just says that you can use EPF, but does not say how to. Other sections, if I'm not wrong, do not add more insights. So, to me this usage remains obscure. Where do I set component's P, Q, U? AFAICT, it is written nowhere, and it is not evident in components' masks.

**Comment by AndreaBartolini** [comment]
You are right, now we have more information about a better strategy to be implemented in order to made less "obscure" the use of the EPF.

Some points should be still discussed (see #151, #152 and #132 ), the new test `PowerGrids.Electrical.Test.SynchronousMachine4WindingsPF2` should be the starting point for the discussion.
